Add john forsyth's blog post about earning bsq

m52go opened the pull request.

Commit:
Add john forsyth's post about earning bsq

ManfredKarrer reviewed (approved):
utACK

Very nice blog post! Congrats!

Commit:
Change tag name to real name

Changed "@arunasurya" to "Aruna Surya" to better fit the medium of a blog post.
